libs/net/models/Areas/Admin/Tags/TagModel.cs

```diff
@@ -12,6 +12,10 @@ public class TagModel : AuditColumnsModel
     /// get/set - The primary key of the type model.
```

Comment:
There is are more than one TagModel classes that need to be updated.  Most endpoint have their own models to ensure they don't result in regression issues when we need to make a change for a specific endpoint.